recipes/aiocontextvars/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "aiocontextvars" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1e0ff5837c8b01c36a1107acdd0baf7853ebdf6c9fc43e8e311f4be37ac2038a
+
+build:
+  number: 0
```

Make this a `noarch` package. The resulting package contents don't depend on the Python version.

```suggestion
  number: 0
  noarch: python
```

---

Thanks for stepping in @xhochy!

Actually, judging from this:

https://github.com/fantix/aiocontextvars/blob/master/setup.py#L14-L16

it doesn't look like a `noarch`, but then inside the package `contextvars` is imported without any condition. Which leads me to wonder if the `setup.py` `install_requires` is somehow wrong.